addons/purchase_stock/models/product.py

```diff
@@ -39,6 +39,7 @@ class ProductProduct(models.Model):
 
     purchase_order_line_ids = fields.One2many('purchase.order.line', 'product_id', string="PO Lines") # used to compute quantities
     monthly_demand = fields.Float(compute='_compute_monthly_demand')
+    actual_demand = fields.Float(compute='_compute_actual_demand')
```

Comment:
I think we can use `outgoing_qty` instead. It's basically the same thing, the only real diff is your compute is simpler than the `outgoing_qty` compute who computes all product's quantities